components/org.wso2.carbon.identity.api.server.idp/org.wso2.carbon.identity.api.server.idp.v1/src/main/java/org/wso2/carbon/identity/api/server/idp/v1/core/ServerIdpManagementService.java

```diff
@@ -2191,6 +2191,10 @@ private JustInTimeProvisioning createJITResponse(IdentityProvider idp) {
                 } else {
                     jitConfig.setScheme(JustInTimeProvisioning.SchemeEnum.PROVISION_SILENTLY);
                 }
+            }
+            if (idp.getJustInTimeProvisioningConfig().getProvisioningUserStore() == null) {
+                jitConfig.setUserstore("PRIMARY");
```

Comment:
Can we use the already defined constant in the carbon core instead of using the string liternal?

Reply:
addressed with [b0d4789](https://github.com/wso2/identity-api-server/pull/363/commits/b0d47895222389aff4b444bb8479246c616ed9d7) in [https://github.com/wso2/identity-api-server/pull/363](https://github.com/wso2/identity-api-server/pull/363)